src/UserStories/UserStories.css

```diff
@@ -157,18 +176,19 @@ html {
 .prev-button {
   grid-area: left;
   width: min-content;
-  margin-left: -50px;
+  margin-left: -46px;
 }
 .next-button {
   grid-area: right;
   width: min-content;
   justify-self: flex-end;
+  margin-right: -46px;
 }
 
 .story-footer {
   z-index: 2;
   position: relative;
-  margin-top: -184px;
+  margin-top: -183px;
```

Comment:
We fixed this by doing -100% I think?

Reply:
Nah, that was a different one.
If you put -100% here, it would end up being in the middle of the story